llvm/test/CodeGen/AMDGPU/invalid-cast-load-i1.ll

```diff
@@ -0,0 +1,43 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx90a %s -o - | FileCheck %s
+
+@g = global ptr null
```

Comment:
I have to make the function have some side effects to be able to generate effective code; otherwise nothing would be generated.

Reply:
Can you name this addrspace(1) or make it a kernel argument